semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -69,6 +69,7 @@ object Main {
   }
 
   private def sinfo(sym: SymbolInfoSymbol): Option[s.SymbolInformation] = {
+    if (sym.parent.get == NoSymbol) return None
```

Comment:
When can this happen?

Reply:
This was the most frequent error, you can uncomment the line and run metacp suite to investigate.